coral-component-colorinput/src/templates/colorLabel.html

```diff
@@ -0,0 +1 @@
+<label handle="colorLabel" type="label"></label>
```

Comment:
Simply including the color value as a block element breaks the layouts of both the ColorInput and ColorInputSwatches. It may be better to use a CoralTooltip to display the value on hover or focus.

```suggestion
<coral-tooltip handle="colorLabel" placement="top" class="_coral-ColorInput-swatch-tooltip"></coral-tooltip>
```